lib/Page.js

```diff
@@ -126,7 +126,9 @@ Page.prototype.cleanUpDependency = function (file) {
 
 /**
  * Pre-render an external dynamic dependency to the same path as the current page
+ * Does not pre-render if file is already pre-rendered by another page during site generation
  * @param file
+ * @param builtFiles, set of files being pre-rendered during a site generation
```

Comment:
"set of files ~`being`~`already` pre-rendered `by another page`~`during a site generation`~"